lms/djangoapps/progress/models.py

```diff
@@ -41,8 +41,9 @@ def get_total_completions(cls, course_key, exclude_users=None, org_ids=None):
             .exclude(user__id__in=exclude_users)
         if org_ids:
             queryset = queryset.filter(user__organizations__in=org_ids)
-        completions = queryset.aggregate(total=Sum('completions'))
-        completions = completions['total'] or 0
+        if group_ids:
+            queryset = queryset.filter(user__groups__in=group_ids).distinct()
+        completions = sum([student_progress.completions for student_progress in queryset])
```

Comment:
There is a bug in older versions of django(< 1.8), Aggregate Sum doesn't respect distinct() in the queryset filter. https://code.djangoproject.com/ticket/24385